### DISCUSSION TITLE
Request Social media Icons

### DISCUSSION THREAD

**Discussion by DrPrime01** [tailwindlabs/heroicons#954] [Ideas]
Social media icons should also be available. I won't like a situation where I have a library that I love for my icons and have to install another library because the former lacks some essential features.

**Comment by sluxzer** [comment]
This is the x times I using heroicons and having the same feeling

**Comment by zefir-git** [comment]
Why burden Heroicons with brands/logos when https://simpleicons.org/ is specifically for this purpose
